Pipfile

```diff
@@ -24,4 +26,4 @@ boto3 = "*"
 appdirs = "==1.4.3"
 
 [requires]
-python_version = "3.6"
+python_version = "3.9"
```

Comment:
ebcli uses some config magic that's not clear to me, but the install platform default is py3.7 - look in .elasticbeanstalk/config.yml

Reply:
anything 3.6+ will let me use f-strings, so that should work

Reply:
According to the elastic beanstalk config, the base image we're using is python3.6 
![image](https://user-images.githubusercontent.com/123998/102911076-8e8f7600-4449-11eb-8a62-6d2fbe23b7b6.png)

So I'll revert this back to 3.6 and make sure the site works with 3.6-3.9